### DISCUSSION TITLE
Using Term.jl in OpenTelemetry.jl

### DISCUSSION THREAD

**Discussion by findmyway** [FedeClaudi/Term.jl#215] [Show and tell]
I'm using Term.jl in [OpenTelemetry.jl](https://github.com/oolong-dev/OpenTelemetry.jl) to render some complex data structures.
The main features I use are `Panel` and `Tree` (with [slight modifications](https://github.com/oolong-dev/OpenTelemetry.jl/blob/master/src/sdk/ext/OpenTelemetrySDKTermExt.jl) ).
It really helps me a lot while debugging. 
By the way, with the new pkg extension feature in Julia@1.9, I no longer need to worry about the large dependency issue when introducing `Term.jl` in my packages ;)

Here are some screen shots:

![image](https://user-images.githubusercontent.com/5612003/231935291-de7fb1a8-5d41-4d62-951d-5fab43db4c91.png)

![image](https://user-images.githubusercontent.com/5612003/231935308-ddd4eb8e-2fcc-4345-af8f-90ba89ef3f5f.png)

**Comment by FedeClaudi** [comment]
Looks awesome!
